### DISCUSSION TITLE
Upload speed do not show

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#2198] [Questions] [answered]
Why is the current file download speed (bitrate) not displayed? Nextcloud v25.0.4

in the case of releases, the bitrate was displayed on the web interface

**Comment by szaimen** [comment]
Hi, what exactly are you not seeing? Can you maybe post a screenshot?

**Reply by ghost** [reply]
![IMG_20180823_162034](https://user-images.githubusercontent.com/126512672/227789788-fe098cbf-3997-449c-af41-a8b2c46cc481.jpg)

**Comment by szaimen** [comment, marked as answer]
Hm, dont know. But this would need to get reported to https://github.com/nextcloud/server/issues in any case.
